spec/factories/user_factory.rb

```diff
@@ -6,5 +6,26 @@
     last_name "Lastly"
     sequence(:email) { |n| "user#{n}@launch.com" }
     password "SuperPWthatislong"
+
+    trait :bot do
+      first_name "Lifehack"
+      last_name "Bot"
+      email "lifehackbot@gmail.com"
+      password "lifehackbot"
+    end
+
+    trait :image do
+      sequence(:first_name) { UserSeeder.fake_name.first }
+      sequence(:last_name) { UserSeeder.fake_name.last }
+      sequence(:email) do |n|
+        "#{UserSeeder.fake_email[1]}#{n}#{UserSeeder.fake_email[2]}"
+      end
+      profile_photo { Rack::Test::UploadedFile.new(File.join(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.